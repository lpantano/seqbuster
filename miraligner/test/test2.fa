AAATGGATTTTTGGAGCAGGGAGATGG	105
AAAACTGTAATTACTTTTGTAC	7
AAATGGATTTTTGGAGCAGG	82389
TACAGTACTGTGATAACTGAA	2
AAAATTCATTGCTGTCGGTG	1
TCCCTGAGACCCTAACTTGTGA	34
GAAAAGCTGGGTTGAGAGGGNNA	34
TAAAGTGCTGACAGTGCAG	35