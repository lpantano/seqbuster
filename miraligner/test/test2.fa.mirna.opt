Initiated at: Thu Mar 28 20:35:35 CET 2013
Mismatches allowed: 1
Addition allowed: 3
Trimming allowed: 3
