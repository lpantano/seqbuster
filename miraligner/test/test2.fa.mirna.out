seq	name	mir	start	end	mism	add	t5	t3	s5	s3	DB	ambiguity
AAAACTGTAATTACTTTTGTAC	seq_2_x7	hsa-miR-548g-3p	54	75	0	0	0	0	TGGCAAAA	GTACCAAC	miRNA	1
AAATGGATTTTTGGAGCAGG	seq_3_x82389	hsa-miR-1246	10	29	1AG	0	qG	0	CTTGAATG	CAGGAGTG	miRNA	1
TACAGTACTGTGATAACTGAA	seq_4_x2	hsa-miR-101-3p	49	69	0	0	0	0	AAGGTACA	TGAAGAAT	miRNA	1
AAAATTCATTGCTGTCGGTG	seq_5_x1	hsa-miR-181b-5p	16	35	3AC	0	0	tGGT	ACTCAACA	GGGTTTGA	miRNA	1
TCCCTGAGACCCTAACTTGTGA	seq_6_x34	hsa-miR-125b-5p	17	38	0	0	0	0	CTAGTCCC	GTGAGGTA	miRNA	1
GAAAAGCTGGGTTGAGAGGGNNA	seq_7_x34	hsa-miR-320a	47	66	0	qNNA	qG	tCGA	CGGGAAAA	GCGAAAAA	miRNA	3
GAAAAGCTGGGTTGAGAGGGNNA	seq_7_x34	hsa-miR-320b	71	90	0	qNNA	qG	tCAA	TGGGAAAA	GCAAAAGG	miRNA	3
GAAAAGCTGGGTTGAGAGGGNNA	seq_7_x34	hsa-miR-320c	30	49	0	qNNA	qG	tT	TGGGAAAA	GGGT	miRNA	3
